src/__tests__/role-helpers.js

```diff
@@ -23,6 +24,27 @@ function setup() {
 
   <nav data-testid='a-nav' />
   
+  <!-- div without an explicit role is given a "generic" role by default -->
```

Comment:
To keep it consistent with the other cases, I wouldn't add a comment here to describe the test because it isn't added for the others.

```suggestion
```

Reply:
I personally find this case more confusing and I'd rather reduce the "Wait what??" moments as much as I can.

I don't mind removing it, just giving my 2 cents :P